commit 0d7b2346729005b74f948c132331ee7d4173785b
Author: Mike Hiretsky <mh@calculate.ru>
Date:   Thu Jul 28 09:31:38 2011 +0400

    Fix the determining of installation image.
    
    LiveUSB excluded from the list of devices from which you can install the
    system if the system installation is on it.

diff --git a/pym/cl_fill_install.py b/pym/cl_fill_install.py
index fdefcbf..030902a 100644
--- a/pym/cl_fill_install.py
+++ b/pym/cl_fill_install.py
@@ -736,9 +736,10 @@ class fillVars(object, glob_attr):
                 livedistr = ['/mnt/cdrom']
         else:
             livedistr = []
+        rootDev = self.Get('os_install_root_dev')
         livedistr += \
             map(lambda x:x[0],
-            filter(lambda x:" live" in x[1],
+            filter(lambda x:" live" in x[1] and x[0] != rootDev,
             zip(self.Get('os_disk_dev'),
                 self.Get('os_disk_content'))))
         return ['/var/calculate/remote/linux',
